WTWSMS/decisions/GR_Scythian.txt

```diff
@@ -203,7 +203,7 @@ decisions = {
 	}
 }
 
-targetted_decisions = {
+decisions = {
```

Comment:
There is no reason to keep those Decisions out of the Decision menu, where they belong.